lms/djangoapps/learner_dashboard/tests/test_utils.py

```diff
@@ -21,3 +29,62 @@ def test_strip_course_id(self, path):
         """
         actual = utils.strip_course_id(path + unicode(utils.FAKE_COURSE_KEY))
         self.assertEqual(actual, path)
+
+
+class DisclaimerIncompleteFieldsTestCase(TestCase):
+    """
+    Test cases to display or not the reminder alert if a user
+    has left some required fields in blank after n days.
+    """
+
+    def setUp(self):
+        self.user = User.objects.create(
+            username='my_self_user',
+            date_joined=date(2016, 3, 1)
+        )
+        self.user_profile = UserProfile.objects.create(
+            user_id=self.user.id,
+            year_of_birth=1989
+        )
+
+    @override_settings(FEATURES={"DAYS_PASSED_TO_ALERT_PROFILE_INCOMPLETION": 5})
+    @override_settings(FEATURES={"FIELDS_TO_CHECK_PROFILE_COMPLETION": ['gender', 'city', 'year_of_birth']})
```

Comment:
If you make these just Site Configuration flags, you can test them using the [`@with_site_configuration`](https://github.com/edx/edx-platform/blob/6d2468d0dcf782a7561ecc61946146d6fae2762f/openedx/core/djangoapps/site_configuration/tests/test_context_processors.py#L30) decorator instead.